library/src/test/java/com/datatorrent/lib/io/block/FSSliceReaderTest.java

```diff
@@ -135,4 +141,32 @@ public void testBytesReceived() throws IOException
     FileUtils.contentEquals(testMeta.dataFile, outputFile);
   }
 
+  @Mock
+  FileSystem fileSystem;
+
+  public class FSTestReader extends FSSliceReader
+  {
+    @Override
+    protected FileSystem getFSInstance() throws IOException
+    {
+      return fileSystem;
+    }
+  }
+
+  @Test
+  public void testBlockSize() throws IOException
+  {
+    long blockSize = 1000;
+    MockitoAnnotations.initMocks(this);
```

Comment:
can you move initMocks to setup method (annotate setup to @BeforeTest) so that every added test case need not do it.